src/Concordium/Client/Runner.hs

```diff
@@ -1269,6 +1295,61 @@ getCryptographicParametersOrDie bhInput = do
         StatusInvalid -> logFatal ["GRPC response contained an invalid status code."]
         RequestFailed err -> logFatal ["I/O error: " <> err]
 
+-- | Compute the time of the first payday after a given time.
+--  This is used for determining the time at which a cooldown will actually elapse.
```

Comment:
Just trying to understand some terminology here: What does "elapse" mean here? Is it different from "expire"? The fact that this takes the expiration time of the cooldown as input suggests it doesn't mean the same thing but I'm not sure what it means in that case. The variable is also called "cooldownEnd" - does a cooldown "end" when it "expires"?